ARCHITECTURE.md

```diff
@@ -0,0 +1,113 @@
+      /grid                               Grid stylesheets
+      /mixins                             Utility mixins
+      /variables                          Globale variables
+```
+
+### Implementatie details
+
+Per component wordt in `readme.md` functionele documentatie bijgehouden.
+
+### Styling
+
+De styling van de toolkit wordt in SCSS geschreven. Vanuit Sources bieden we de globale (basis) en component styling aan.
+
+Als een component meerdere implementaties krijgt centraliseren we de styling in deze `sources` package en bieden we die aan als Sass mixins. Als een component maar één implementatie krijgt is de styling alleen in de betreffende package te vinden. Uiteraard baseert de styling zich wel op variables die in `sources` worden gezet.
+
+### Component model
+
+Per component is er een model (TypeScript interface) waar de functionaliteit in code wordt vastgelegd. Dit model is het contract waar implementaties aan moeten voldoen. Dit is een implementatie- en Storybook agnostisch model.
+
+### Storybook definities
+
+Per component wordt de Storybook definitie bijgehouden. In de basis is dit een `export function storiesOfComponent(storybookParameters, componentParameters)`:
+
+* `storybookParameters` is een component-overstijgend model voor de basis functionaliteit van Storybook.
+* `componentParameters` is een component specifiek model. Over het algemeen zal dit alleen om de implementatie-specifieke template van het component gaan, maar voor complexe werkvormen kan dit uitgebreid worden om de documentatie te verbeteren.
+
+Met deze opzet worden implementaties geforceerd identieke varianten op te leveren.
+
+### Storybook args
+
+Vanuit Storybook kan een component worden bediend en content worden aangeboden. Deze functie is het koppelvlak tussen demo/dummy content uit `<component>.content.ts` en Storybook Controls. De mapper `function componentArgsMapper(args: componentArgs)` kan voor simpele werkvormen soms overbodig voelen. Mochten de `ComponentModel` en `ComponentArgs` overeenkomen kan er gebruik gemaakt worden van de object spread shorthand `return { ...args };`.
+
+## Storybook
+
+De diverse implementaties van de DSO Toolkit componenten worden ieder in een eigen Storybook getoond.
+
+### HTML/CSS components
+
+De HTML/CSS componenten worden met `lit-html` gedocumenteerd.
+
+### Web Components
+
+Web Components schrijven we in TypeScript met [Stencil](https://stenciljs.com/).
+
+### React Components
+
+React Components worden gegenereerd door Stencil. Voor elke Web Component wordt er een binding component gegenereerd waarmee de React developer (afnemer) op een vertrouwde manier met de DSO Toolkit Web Components kan werken.
+
+## Fractal
+
+Fractal is het documentatie framework waar we mee zijn begonnen. Dit framework is niet meer ondersteund en werkt steeds meer tegen ons dan voor ons. Templates worden in Nunjucks geschreven waardoor het lastig is om componenten te "slotten".
```

Comment:
'werkt steeds meer tegen ons dan voor ons' is zeker waar, maar hoeft geen plek te krijgen in een document als dit. Ik zou simpelweg stellen dat "Dit framework is end-of-life en zal worden deprecated".